src/main/java/core/Main.java

```diff
@@ -50,5 +56,22 @@ public static void main(String[] args) {
                 .forEach(System.out::println);
     
         System.out.println("\n=============================\n");
+        
+        User user = new User();
+        user.setEmail("user.email@email.com");
+        user.setPassword("strong_password");
```

Comment:
it is redundant to set these values to the user here

Reply:
sure